### DISCUSSION TITLE
HA - Santization start

### DISCUSSION THREAD

**Discussion by djgel** [Yogui79/IntexPureSpa#23] [Q&A] [answered]
Hi, 

If put  on a  switch.intex_sanitizer, its return to Off, filter is same  .... i imagine  because a   set time , how to make this ?

**Comment by LEECHER1** [comment, marked as answer]
The Sanitizer time use this Command before you turn on the Sanitizer Button. 

Sanitizer Timer | IntexSpa/Cmd Sanitizer time | hours=3,5,8 | - | -
-- | -- | -- | -- | --

**Reply by LEECHER1** [reply]
See my last Video. To see it how it works.

**Reply by djgel** [reply]
@LEECHER1 , thks  for attention! there was an error in the switch! now it works! Start, stop wait time confirm and start again. Now make  a  scheduler to star 3 timesfor week ! Thks

**Reply by LEECHER1** [reply]
Good :) 

i have Scheduler for 1 h (1h with Auto off) per day. I check it out it is enough chlorine.

How long to you starting?

**Comment by djgel** [comment]
I thought and set it up by activating it (3 hours activated by the pump pattern) at 11:59PM tuesday, thursday saturday and sunday (Days that have more use ... more time with it uncapped providing greater chlorine evaporation) I have to, in Mondays, Wednesdays and Fridays only enable filtering. Active heating when I plan to enter in advance. With HA, and since I use Nabu Casa, I can activate remotely now!

**Reply by djgel** [reply]
can you publish a  input.selection use por  set time  for filter and sanitization  ?

**Reply by LEECHER1** [reply]
Hey,

Yes but this not a tutorial to copy and paste.

You need by the sanitizer a dropdown menu the slider is not working. By the filter you can use a slider or dropdown. You see only the dropdown version.

It is all in German.



1. configuration.yaml


  - platform: mqtt
    name: "IntexSpa/Filter Setup Zeit" #change the name do you what
    state_topic: "IntexSpa/filter setup time"
  - platform: mqtt
    name: "IntexSpa/Desinfektion setup Zeit" #change the name do you what
    state_topic: "IntexSpa/Sanitizer setup time"

--------------

This is the helper function
(Configuration -> Helpers)

See the Screenshot.

![Helper-Dropdown](https://user-images.githubusercontent.com/12032696/127025963-67a771d1-6176-4f38-bed8-ac32b5dffbdf.PNG)

-------------
3. automations.yaml

Three version Automations 3h, 5, 8h

- id: '1620839242388'
  alias: intex Desinfektion Dropdown - 3
  description: ''
  trigger:
  - platform: numeric_state
    below: '4'
    entity_id: input_select.intex_desinfektion_dropdown
  condition: []
  action:
  - service: mqtt.publish
    data:
      topic: IntexSpa/Cmd Sanitizer time
      payload: '3'
  mode: single
- id: '1620839267752'
  alias: intex Desinfektion Dropdown - 5
  description: ''
  trigger:
  - platform: numeric_state
    below: '6'
    entity_id: input_select.intex_desinfektion_dropdown
    above: '4'
  condition: []
  action:
  - service: mqtt.publish
    data:
      topic: IntexSpa/Cmd Sanitizer time
      payload: '5'
  mode: single
- id: '1620839288864'
  alias: intex Desinfektion Dropdown - 8
  description: ''
  trigger:
  - platform: numeric_state
    below: '9'
    entity_id: input_select.intex_desinfektion_dropdown
    above: '7'
  condition: []
  action:
  - service: mqtt.publish
    data:
      topic: IntexSpa/Cmd Sanitizer time
      payload: '8'
  mode: single


And Screenshot for one automation:
![Auto](https://user-images.githubusercontent.com/12032696/127027116-3d67ab67-1c91-4782-945b-a53258ce6b93.png)


I hope i can help you.

**Reply by djgel** [reply]
Change only de  input_selection for  Input_number! Thanksssssss !

**Reply by LEECHER1** [reply]
> Change only de input_selection for Input_number! Thanksssssss !

sorry, what do you mean?

**Reply by djgel** [reply]
Sorry, just meant I changed Input_selection to Input_number or "helper" to use sliding selection. Everything works 100% . Thank you
